ops-bedrock/entrypoint-l1.sh

```diff
@@ -27,6 +27,10 @@ fi
 if [ ! -d "$GETH_CHAINDATA_DIR" ]; then
 	echo "$GETH_CHAINDATA_DIR missing, running init"
 	echo "Initializing genesis."
+	# Handle setting the clique config in the extra data
+	EXTRA_DATA="$BLOCK_SIGNER_ADDRESS""0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
This needs the clique config to be set as well

Reply:
Should there then be a check here that the "0x" prefix is _not_ prepended to the extra data? 

The removed go code would do that through its use of common.Address as I understand it.

Reply:
you are right that its a bug there is no `0x` prefix